Automattic/woocommerce-payments
# Fix builders being stuck after leaving KYC without finishing

Fixes #7626

#### Changes proposed in this Pull Request

Redirect merchants from `payments/onboarding` to `payments/overview` if merchant has Stripe account connected and somehow got to this page (see steps below) and the video in the issue.

#### Testing instructions

* Ensure Progressive Onboarding feature flag is enabled
* Onboard a builder account
* On Payments -> Overview page, click Set up live payments
* Pick builder mode again, click continue
* Get redirected to Stripe
* Click back button in the browser
* Ensure that you are redirected to the payments overview page and you don't see Not allowed error

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-6.8.0#fix-builders-being-stuck-after-leaving-kyc-without-finishing_
